content/shmem_init_thread.tex

```diff
@@ -0,0 +1,53 @@
+\apisummary{
+Initializes the OpenSHMEM library, similar to \FUNC{shmem\_init}, and performs any
+initialization required for supporting the four thread levels.}
+
+%Initializes the \openshmem{} library, similar to \FUNC{shmem\_init}, and in addition performs the initialization required for thread-safe invocation of \openshmem{} functions.}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+int shmem_init_thread(int requested, int *provided);
+\end{Csynopsis}
+
+\begin{apiarguments}
+\apiargument{IN}{requested}  {The thread level support requested by the user.
+}
+\apiargument{OUT}{provided}{The thread level support provided by the \openshmem{} implementation.}
+\end{apiarguments}
+
+
+\apidescription{
+\FUNC{shmem\_init\_thread}  initializes the \openshmem{} library similar to
+\FUNC{shmem\_init}, and in addition performs the initialization required for
+supporting the four thread levels. The argument
+\VAR{requested} is used to specify the desired level of thread support.
+The argument \VAR{provided} returns the support level provided by the library.
+The correct values for \VAR{provided} and \VAR{requested} are 
+\CONST{SHMEM\_THREAD\_SINGLE}, \newline 
+\CONST{SHMEM\_THREAD\_FUNNELED}, \CONST{SHMEM\_THREAD\_SERIALIZED}, or 
+\CONST{SHMEM\_THREAD\_MULTIPLE}.
+     
+An \openshmem{} program is initialized either by \FUNC{shmem\_init} or
+\FUNC{shmem\_init\_thread}. Similar to \FUNC{shmem\_init}, the
+\FUNC{shmem\_init\_thread} may not be called multiple times in an \openshmem{} program.
+ If the call to \FUNC{shmem\_init\_thread}  is unsuccessful either in allocating and initializing
+ resources for the \openshmem{} library or initialization required for supporting
+ the provided thread level, then the behavior of any subsequent call
+ to the \openshmem{} library is undefined.
+}
```

Comment:
Strike: "or initialization required for supporting the provided thread level"

Reply:
Would you be ok with doing this edit through the section committee?  We need to prioritize merging this PR, so the section committees can get to work on it.

Also, Manju, please hard wrap these lines.

Reply:
@jdinan This particular edit was already handled and read.